SpatialGDK/Source/SpatialGDK/Private/Utils/SpatialDebugger.cpp

```diff
@@ -259,17 +260,75 @@ void ASpatialDebugger::OnEntityRemoved(const Worker_EntityId EntityId)
 	EntityActorMapping.Remove(EntityId);
 }
 
+void ASpatialDebugger::ActorAuthorityChanged(const Worker_AuthorityChangeOp& AuthOp) const
+{
+	const bool bAuthoritative = AuthOp.authority == WORKER_AUTHORITY_AUTHORITATIVE;
+
+	if (bAuthoritative && AuthOp.component_id == SpatialConstants::AUTHORITY_INTENT_COMPONENT_ID)
+	{
+		if (NetDriver->VirtualWorkerTranslator == nullptr)
+		{
+			// Currently, there's nothing to display in the debugger other than load balancing information.
+			return;
+		}
```

Comment:
Both AuthorityIntent component and VirtualWorkerTranslator VirtualWorkerTranslator should only exist if load balancing is enabled, when would this if get hit?

Reply:
I see SpatialDebugger as a much more flexible long term tool, so I could definitely see us or developers adding more information to it in the long term. Even if we don't have that now, I added this in a way that it could be added easily in the future.